ghost/core/core/frontend/helpers/tpl/content-cta.hbs

```diff
@@ -1,19 +1,31 @@
 {{{html}}}
 <aside class="gh-post-upgrade-cta">
-    <div class="gh-post-upgrade-cta-content" style="background-color: {{accentColor}}">
+    <div class="gh-post-upgrade-cta-content" style="background-color: {{@site.accent_color}}">
         {{#has visibility="paid"}}
-            <h2>This post is for paying subscribers only</h2>
+            {{#is "page"}}
```

Comment:
Is there a reason for not doing this inline? e.g. `<h2>This {{#is "page"}}page{{else}}post{{/is}} is for paying subscribers only</h2>`

Reply:
Will switch to this, no reason other than this one felt cleaner to me, but no strong prefs. :)